sqlflow_models/dnnclassifier.py

```diff
@@ -1,7 +1,7 @@
 import tensorflow as tf
 
 class DNNClassifier(tf.keras.Model):
-    def __init__(self, feature_columns=None, hidden_units=[10,10], n_classes=3):
+    def __init__(self, feature_columns=None, hidden_units=[100,100], n_classes=3):
```

Comment:
Why updating default values?

Reply:
1. To make the unit test stably meet the loss criterion.
1. A network of `[10, 10]` is less than useless for almost all real-world datasets.

Reply:
I see. Thanks.